src/DaPigGuy/PiggyCustomEnchants/enchants/weapons/LightningEnchant.php

```diff
@@ -21,14 +21,6 @@ class LightningEnchant extends ReactiveEnchantment
     /** @var string */
     public $name = "Lightning";
 
-    /**
-     * @return array
-     */
-    public function getReagent(): array
-    {
-        return [EntityDamageByEntityEvent::class];
-    }
-
```

Comment:
This class should ``use ReactiveTrait`` instead of extending ReactiveEnchantment
> Couldn't comment line 19, so I commented this line